UI/Console.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#ifndef CONSOLE_H
+#define CONSOLE_H
+
+#include "imgui.h"
+#include <vector>
+#include <string>
+
+
+
+void consoleLog(std::string str);
+void drawConsole();
+
+#endif
```

Comment:
Conversation: It looks like you are making all the "components" in a static way, do you foresee needing to make them "objects" of any kind in the future? aka they might store a lot of information, or do you foresee the UI team just reading stuff from elsewhere in a static way?

Reply:
I'm assuming that not much information will be stored and most will be accessed from other places, so it works like this right now, though if you think it would be better to make classes then I can change them to classes